backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/DockerRabbitMQ.java

```diff
@@ -79,10 +81,13 @@ private DockerRabbitMQ(Optional<String> hostName, Optional<String> erlangCookie,
         this.nodeName = nodeName;
     }
 
-    private WaitAllStrategy waitStrategy() {
+    private WaitStrategy waitStrategy() {
         return new WaitAllStrategy()
-            .withStrategy(Wait.forHttp("").forPort(DEFAULT_RABBITMQ_ADMIN_PORT).withRateLimiter(RateLimiters.TWENTIES_PER_MINUTE))
-            .withStrategy(new RabbitMQWaitStrategy(this, Duration.ofMinutes(10)));
+            .withStrategy(Wait.forHttp("").forPort(DEFAULT_RABBITMQ_ADMIN_PORT)
+                .withRateLimiter(RateLimiters.DEFAULT)
+                .withStartupTimeout(TEN_MINUTE_TIMEOUT))
+            .withStrategy(new RabbitMQWaitStrategy(this, TEN_MINUTE_TIMEOUT))
+            .withStartupTimeout(TEN_MINUTE_TIMEOUT);
```

Comment:
spotted the bottle neck. Tests still fail when specified 10 minutes for each wait strategy, problem came from the composer `WaitAllStrategy` has it own default timeout = 30s. We need to specify timeout to it also.